pkg/controller/compliancescan/config.go

```diff
@@ -57,8 +57,7 @@ cmd=(
     oscap-chroot $HOSTROOT xccdf eval \
     --fetch-remote-resources \
     --profile $PROFILE \
-    --report /tmp/scan-report.xml \
-    --results-arf /tmp/report.xml
+    --results-arf $ARF_REPORT
```

Comment:
stupid question, what's the difference between the report and the results-arf?

Reply:
--report is the XCCDF file that contains the selected tests and their results. For our clusters and the content we have it's typically less than a megabyte. OTOH --results-arf is the data stream with all the checks and remediations and the XCCDF together.
What would be better names? Would it be better to call them `--test-results` and `--arf-report`?

Reply:
Would you like me to rename the command line switches?

Reply:
up to you, I don't have a strong opinion about this.

Reply:
Ooh, only now did I notice that this is about the arguments for openscap, not for the pod. OK, in that case we can't remove the arguments :-) As per why didn't I use the output provided by `--report` in the first place and why I used the ARF -- the report still includes some data from the DS which we don't need. The rds-split was something Simon suggested on chat and only includes the rule results.